Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -179,6 +186,10 @@ extension CommentRepliesViewController {
       self.viewModel.inputs.didSelectComment(comment)
     }
   }
+
+  override func tableView(_: UITableView, willDisplay cell: UITableViewCell, forRowAt _: IndexPath) {
```

Comment:
Huh, I'm guessing this is old code, but I'm surprised it's setting up a cell delegate instead of just implementing `tableView:didSelectCell:atIndexPath`